### PR TITLE
RavenDB 13962 - Force Revisions Configuration

### PR DESCRIPTION
**Pull request by Danielle9897**

### CODE REVIEW

File: src/Raven.Studio/typescript/commands/database/settings/enforceRevisionsConfigurationCommand.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    execute(): JQueryPromise<operationIdDto> {
+        const url = endpoints.databases.revisions.adminRevisionsConfigEnforce;       
+
+        return this.post<void>(url, null, this.db)
+            .fail((response: JQueryXHR) => {
+                this.reportError("Failed to enforce revisions configuration", response.responseText, response.statusText); 
+            })
+            .done(() => {
+                this.reportSuccess("Revisions configuration enforced successfully");
```
Comment (ml054) [inline src/Raven.Studio/typescript/commands/database/settings/enforceRevisionsConfigurationCommand.ts:19]: please check if this is sync or async operation (in handler) - if async then we should write that config was enforced.

**Review by ml054 — CHANGES_REQUESTED** [review]
please check my single comment in code

File: src/Raven.Studio/typescript/commands/database/settings/enforceRevisionsConfigurationCommand.ts (hunk 1 above)
Comment (Danielle9897) [reply src/Raven.Studio/typescript/commands/database/settings/enforceRevisionsConfigurationCommand.ts:19]: `EnforceConfigRevisions` operation in handler is **sync** - so I assume it is ok like this.

Comment (ml054) [reply src/Raven.Studio/typescript/commands/database/settings/enforceRevisionsConfigurationCommand.ts:19]: ok great - I was just double checking.

**Review by ml054 — APPROVED** [review]

**Pull request merged**